src/app/components/RuntimeEvents/__tests__/matchEventAndTokenTransfersIcons.test.tsx

```diff
@@ -3,17 +3,17 @@ import { EventTypeIcon } from '../RuntimeEventDetails'
 import { TokenTransferIcon } from '../../Tokens/TokenTransferIcon'
 
 test('Transfer, burn, and mint icons should match in EventTypeIcon and evm TokenTransferIcon', () => {
-  const transfer = render(<EventTypeIcon eventType="accounts.transfer" eventName="Transfer" />).container
-  const evmTransfer = render(<TokenTransferIcon name="Transfer" size={25} />).container
+  const transfer = render(<EventTypeIcon eventType="accounts.transfer" />).container
+  const evmTransfer = render(<TokenTransferIcon method="Transfer" size={25} />).container
   expect(transfer.querySelector('svg')?.outerHTML.length).toBeGreaterThan(100)
   expect(transfer.querySelector('svg')?.outerHTML).toEqual(evmTransfer.querySelector('svg')?.outerHTML)
 
-  const mint = render(<EventTypeIcon eventType="accounts.mint" eventName="Tokens minted" />).container
-  const evmMint = render(<TokenTransferIcon name="Minting" size={25} />).container
+  const mint = render(<EventTypeIcon eventType="accounts.mint" />).container
+  const evmMint = render(<TokenTransferIcon method="Minting" size={25} />).container
```

Comment:
Translation for _Minting_ missing?

Reply:
This is a test file so there is no need to use translation hooks. Other than that,
these are not labels as long as I understand https://github.com/oasisprotocol/explorer/commit/29e8605c1. I don't have insights why we did it this way.

Reply:
Got it. My bad 😄